ADVANCED.md

```diff
@@ -20,9 +26,13 @@ __Example :__
       server_mounted_mirror = var.MIRROR
     }
 ```
- - variable.tf file, remove all the variables from the main.tf and move it to a specific file
+
+### variable.tf
+
+File, remove all the variables from the main.tf and move it to a specific file
```

Comment:
Proper naming of custom things should be in `this`, otherwise some linters may fail if non-English word is not found in spelling.
```suggestion
### `variable.tf` file

Remove all the variables from the `main.tf` and move it to a specific file
```

Reply:
Nice catch, thank you.